community/cypher/cypher-compiler-3.1/pom.xml

```diff
@@ -22,7 +22,7 @@
   </scm>
 
   <properties>
-    <version-package>cypher.internal.compiler.v3_0</version-package>
+    <version-package>cypher.internal.compiler.v3_1</version-package>
```

Comment:
It seems to come from https://github.com/neo4j/neo4j/commit/219fbf033de8914f1fefc5e362a694778f421393, but perhaps this is no longer used.